src/test/java/com/urswolfer/gerrit/client/rest/http/projects/ProjectsParserTest.java

```diff
@@ -78,4 +84,17 @@ public void testParseSingleProjectInfo() throws Exception {
             .withState(ProjectState.ACTIVE)
             .get());
     }
+
+    @Test
+    public void testGenerateProjectInput() throws Exception {
+        ProjectInput projectInput = new ProjectInput();
+        projectInput.name = "MyProject";
+        projectInput.description = "This is a demo project.";
+        projectInput.submitType = SubmitType.CHERRY_PICK;
+        projectInput.owners = Collections.singletonList("MyProject-Owners");
+
+        String actualProjectInput = Files.readFile(getFile("project_input.json")).replaceAll("\n","");
+        Truth.assertThat(projectsParser.generateProjectInput(projectInput))
```

Comment:
Ah, you have added to JSON conversion to the parser for easier testability?

Don't you think comparing a plain test string could be unsafe (probably because of changed order of properties in objects or whitespace changes in another version of Gson)?

I think it would also be OK to just test it in the ProjectsRestClientTest like you have done it already.